package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cabal-core",
-  "version": "3.0.4",
+  "version": "3.1.0",
```

Comment:
Could we do version bumps separately from PRs?

Reply:
lol yeah sorry, i keep thinking it's a good place to do in PRs. fixing!